Dockerfile

```
@@ -0,0 +1,11 @@
+FROM ruby:3.2
```

Lets keep the image and small as possible

```suggestion
FROM ruby:3.2-slim

RUN apt-get update -qq && \
    apt-get install -y \
    build-essential \
    git \
    && rm -rf /var/lib/apt/lists/*
```